crm_timesheet/views/hr_timesheet_view.xml

```diff
@@ -11,6 +11,7 @@
                     name="lead_id"
                     invisible="context.get('base_model_name') == 'crm.lead'"
                     groups="sales_team.group_sale_salesman"
+                    optional="hide"
```

Comment:
To keep the status quo:

```suggestion
                    optional="show"
```

Reply:
Thanks @pedrobaeza , 
You're right, statu quo is better.
Is there any persistence of the conf per user ?

Reply:
Odoo stores the shown/hidden columns by browser.